File: aurafit-frontend/src/Models/DTOS/ProductCreateDTO.ts
import {ProductType} from "../ProductType.ts";
import {Category} from "../Category.ts";
import {Gender} from "../Enums/Gender.ts";


export class ProductCreateDTO {
    name: string;
    description: string;
    gender: Gender;
    category: Category;
    productType: ProductType;


    constructor(name:string ,description: string, gender: Gender  ,category: Category ,productType: ProductType) {
        this.name = name;
        this.description = description;
        this.gender = gender;
        this.category = category;
        this.productType = productType;
    }
}

File: AuraFit-BackEnd/src/main/java/app/aurafitbackend/product/ProductCreateDTO.java
package app.aurafitbackend.product;

import app.aurafitbackend.category.Category;
import app.aurafitbackend.Enums.Gender;
import app.aurafitbackend.productType.ProductType;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class ProductCreateDTO {

    private String name;
    private String description;
    private Gender gender;
    private Category category;
    private ProductType productType;

}
